mojo/public/cpp/platform/tcp_platform_handle_utils_posix.cc

```diff
@@ -120,7 +120,18 @@ base::ScopedFD CreateTCPServerHandle(size_t port) {
     PLOG(ERROR) << "listen" << handle.get();
     return base::ScopedFD();
   }
-  LOG(INFO) << "Server Sock fd for port " << port <<":"<<handle.get();
+
+  // Get port number
+  if (port == 0) {
+    struct sockaddr_in sin;
+    socklen_t len = sizeof(sin);
+    if (getsockname(handle.get(), (struct sockaddr*)&sin, &len) < 0)
+      PLOG(ERROR) << "getsockname() " << handle.get();
```

Comment:
`handle` will be returned although `getsockname` is failed. Is it OK?

Reply:
Done. return empty handle when getsockname is failed.